types/rascal/index.d.ts

```diff
@@ -165,6 +165,11 @@ export interface Recovery {
     immediateNack?: boolean | undefined;
 }
 
+export interface ConnectionInfo {
```

Comment:
I would rename this to ConnectionDetails to match the [code](https://github.com/onebeyond/rascal/blob/master/lib/amqp/Vhost.js#L188)